wiki/hlsguide

```diff
@@ -0,0 +1,147 @@
+# Setting Up and Using Haskell Language Server in VS Code
+
+## Introduction
+
+The [Haskell Language Server (HLS)](https://github.com/haskell/haskell-language-server) is a powerful tool that provides IDE-like features for Haskell development. 
+It integrates directly with VS Code through the [Haskell extension](https://marketplace.visualstudio.com/items?itemName=haskell.haskell), enabling features such as code completion, error diagnostics, refactoring, and more. 
+This guide covers setting up HLS in VS Code specifically for Courseography and highlights the most useful features to make the developer experience more enjoyable. :)
+
+## Installation
```

Comment:
So this is actually a case where I'd like to see you move this section to the README. Add it to the section for "Developers".

On this page, you can just add a note in the intro that developers should install the extension by following the instructions in the readme.